Prospress/action-scheduler
# needs refresh update composer.json

Fixes #422 

This PR updates `composer.json` to

- specify minimum PHP version
- specify default PHP version
- update PHPUnit to 7.5
- update woocommerce sniffs

### Testing

- Run `composer install`
- Run unit tests
- Run `vendor/bin/phpcs classes` 
- phpcs should generate lots of messages
